packages/api-db-mongodb/src/migration.ts

```diff
@@ -628,8 +628,13 @@ export const Migrations: Migration[] = [
         const paymentProvidersCustomersArray: PaymentProviderCustomer[] = []
         const paymentProviderCustomers = Object.keys(user.paymentProviderCustomers)
         paymentProviderCustomers.forEach(ppc => {
-          // @ts-ignore
-          const userPPC = user.paymentProviderCustomers[ppc]
+          interface OldPaymentProviderCustomer extends PaymentProviderCustomer {
+            customerID: never
+            id: PaymentProviderCustomer['customerID']
+          }
+
+          const userPPC = user.paymentProviderCustomers[+ppc] as OldPaymentProviderCustomer
```

Comment:
as `paymentProviderCustomers` is an array, `ppc` has to be a number else the typing will be `any`, `+` casts the string to a number (also in JavaScript).